app/Http/Livewire/Airfields/Index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Livewire\Airfields;
+
+use Illuminate\Support\Facades\Http;
+use Livewire\Component;
+
+// TODO: move to outside controller when need arises.
+class Airfield
```

Comment:
Are we going to move this outside now? Would have thought it best to start as we mean to go on.
A Laravel API Resource might work here (`UKCPAirfieldResource::collection($airfield)`) - that being said not sure if it gives us any benefits. Could possibly make our own extension of the `Collection` class (e.g. `AirfieldCollection`) so that the "search" function can be a function on the collection

Reply:
Good call. I didn't think about using them to reverse-engineer a a consumed API. Always thought of them as a first-party thing. Take a look in the latest changeset.